stable/docker-registry/templates/deployment.yaml

```diff
@@ -69,6 +69,14 @@ spec:
                 secretKeyRef:
                   name: {{ template "docker-registry.fullname" . }}-secret
                   key: s3SecretKey
+            - name: REGISTRY_STORAGE_S3_REGION
+              value: {{ required ".Values.s3.region is required" .Values.s3.region }}
```

Comment:
Unfortunately, required values won't work with our CI. The job will fail because there is no way to set them.

Reply:
This path only applies when `storage: s3`, which is not the default.